tests/ctst/features/iam-policies/AssumeRoleWithWebIdentity.feature

```diff
@@ -125,52 +125,52 @@ Feature: Assume Role with Web Identity
-      | GetObjectLockConfiguration      | DATA_CONSUMER         | without        | exists         | ObjectLockConfigurationNotFoundError        |
-      | GetObjectRetention              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | GetObjectLegalHold              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | PutObjectRetention              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | PutObjectLegalHold              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | GetBucketReplication            | DATA_CONSUMER         | without        | does not exist | ReplicationConfigurationNotFoundError       |
-      | GetBucketLifecycleConfiguration | DATA_CONSUMER         | without        | does not exist | NoSuchLifecycleConfiguration                |
-      | GetObjectRetention              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | GetObjectLegalHold              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | PutObjectRetention              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | PutObjectLegalHold              | DATA_CONSUMER         | with           | exists         | Bucket is missing Object Lock Configuration |
-      | GetBucketTagging                | DATA_CONSUMER         | without        | does not exist | NoSuchTagSet                                |
-      | PutObjectLockConfiguration      | DATA_CONSUMER         | without        | exists         | InvalidBucketState                          |
+      | action                          | type                  | withVersioning | objectExists   | error                                           |
+      | RestoreObject                   | STORAGE_MANAGER       | with           | exists         | InvalidObjectState                              |
+      | GetBucketCors                   | STORAGE_MANAGER       | without        | does not exist | NoSuchCORSConfiguration                         |
+      | GetObjectLockConfiguration      | STORAGE_MANAGER       | without        | exists         | ObjectLockConfigurationNotFoundError            |
+      | GetObjectRetention              | STORAGE_MANAGER       | with           | exists         | Bucket is missing Object Lock Configuration     |
+      | GetObjectLegalHold              | STORAGE_MANAGER       | with           | exists         | Bucket is missing Object Lock Configuration     |
+      | PutObjectRetention              | STORAGE_MANAGER       | with           | exists         | MalformedXML: request xml is undefined or empty |
```

Comment:
nit: for consistency with the other expected errors, should we only specify the error type `MalformedXML` without the description ?